libff/algebra/curves/bls12_377/bls12_377.sage

```diff
@@ -175,23 +226,26 @@ def g2_fast_cofactor_coefficients(curve_order, q, h2):
     See Section 3.1: https://eprint.iacr.org/2017/419.pdf
 
     Multiplication by 3*h2 of P in E'/F_{q^2} can be reduced to:
-      [x^3−x^2−x+4]P+ [x^3−x^2−x+1]ψ(P) + [−x^2+2x−1]ψ^2(P)
+      [x^3-x^2-x+4]P + [x^3-x^2-x+1]ψ(P) + [-x^2+2x-1]ψ^2(P)
     """
 
     # For P in E'(Fq2),
     #   [q]P = [t]ψ(P) - ψ^2(P)
+    # where t is the trace of Frobenius, which Hasse tells us is s.t.
```

Comment:
This makes the sentence very long and difficult to parse.  Perhaps we can move this to the start of the paragraph: `Let t be the trace of Frobenius (which, satisfies ....).  Then, for P in E'(Fq2),`.

Reply:
Thanks. I rephrased in the last commit. Let me know what you think

Reply:
Looks good.  Thanks.